test/hyperswitch/filters.js

```diff
@@ -5,6 +5,14 @@ var Server = require('../utils/server.js');
 var preq   = require('preq');
 var P      = require('bluebird');
 
+function range(n) {
+    var a = new Array(n);
+    for (var i = 0; i < n; i++) {
+        a[i] = i;
+    }
+    return a;
+}
```

Comment:
That's less obvious, and also likely slower. It does win on lines of code, though ;)